src/frontend/js/components/PaymentButton/components/PaymentInterfaces/LyraPopIn.tsx

```diff
@@ -0,0 +1,121 @@
+import KRGlue from '@lyracom/embedded-form-glue';
+import { useIntl } from 'react-intl';
+import { useRef } from 'react';
+import * as Joanie from 'types/Joanie';
+import { PaymentErrorMessageId } from 'components/PaymentButton/index';
+import { useAsyncEffect } from 'hooks/useAsyncEffect';
+import { handle } from 'utils/errors/handle';
+
+const LYRA_FORM_ID = 'lyra-form';
+
+const LyraPopIn = ({
+  onSuccess,
+  onError,
+  configuration,
+  form_token,
+}: Joanie.PaymentInterfaceProps<Joanie.LyraPayment>) => {
+  const intl = useIntl();
+  const shouldAbort = useRef<Boolean>(true);
+
+  const handleError = (error?: Error) => {
+    if (error) handle(`[LyraPopIn] - ${error}`);
+
+    if (shouldAbort) {
+      onError(PaymentErrorMessageId.ERROR_ABORTING);
+    }
+    onError(PaymentErrorMessageId.ERROR_DEFAULT);
+  };
+
+  const initializeLyraForm = async () => {
+    const { base_url: baseUrl, public_key: publicKey } = configuration;
+
+    const link = document.createElement('link');
+    link.setAttribute('rel', 'stylesheet');
+    link.setAttribute('href', `${baseUrl}/static/js/krypton-client/V4.0/ext/neon-reset.min.css`);
```

Comment:
Couldn't we use settings for this URL?

Reply:
IMO it's really specific to this component and we do not want to customize it so I'll do not change that until we need it.